### PR TITLE
closes #65

### LINKED ISSUE #65: Add `dividends_annual` property for Portfolio
`dividends_annual` property [exists](https://github.com/mbk-dev/okama/blob/90e25cb4f8f7e93578a2ae33068c7fc0fa291d1d/okama/asset_list.py#L861) in AsssetList. The same should be in Portfolio.

### PR DESCRIPTION
**Pull request by JVthearchitect**
add dividends_annual property to Portfolio
includes test file "t_test_portfolio_dividends_annual.py"

Please let me know if there are any errors or follow ups.

### CODE REVIEW

**Comment by JVthearchitect** [comment]
It looks like I submitted this twice. I did not mean to do that. Sorry for the confusion.

File: t_test_portfolio_dividends_annual.py
Comment (chilango74) [inline t_test_portfolio_dividends_annual.py:1]: There are tests for Portfolio in [test_portfolio.py](https://github.com/mbk-dev/okama/blob/master/tests/test_portfolio.py). Tests case are already defined in [conftest.py](https://github.com/mbk-dev/okama/blob/master/tests/conftest.py). Please use one of them. And move the test to test_potfolio.py.

Comment (JVthearchitect) [reply t_test_portfolio_dividends_annual.py:1]: Thanks for your feedback. I really love this project and this is my first open-source contribution. Please let me know if this is not what you are looking for. I moved the tests to test_portfolio.py and utilized the portfolio provided in conftest.py. My tests passed so this should be good. I submitted a new pull request.

Comment (chilango74) [reply t_test_portfolio_dividends_annual.py:1]: Hi, @JVthearchitect 
Thank you for contributing! It's good for the first PR. The main part is done well. However, in the test is still missing the main part. The test must compare the value with the expected output. You should use `assert` statement for this. In this case the new property returns time series. Please see like it's done in this [example](https://github.com/mbk-dev/okama/blob/90e25cb4f8f7e93578a2ae33068c7fc0fa291d1d/tests/test_portfolio.py#L299C46-L299C46).

**Pull request closed without merging**